src/views/players/EditProfile/components/EditPlayerAboutMe.vue

```diff
@@ -28,16 +28,17 @@
   import EditPlayerFeaturedAchievement from './EditPlayerFeaturedAchievement.vue';
   import EditPlayerNewSection from './EditPlayerNewSection.vue';
 
+
   @Component({
     components: { EditPlayerFeaturedAchievement, EditField, EditPlayerNewSection },
   })
   export default class PlayerAboutMe extends Vue {
-    get user() {
-      return this.$vxm.user.userDetails;
+    get user(): UserData {
+      return this.$vxm.user.userDetails as UserData;
```

Comment:
`... as UserData` -> `...!`

Reply:
With the amount of times I see this line in the code, I'm thinking that either there should be a runtime null check here, or that we should change the type that's in the store.
It's probably out of scope for this PR though.

Reply:
Should definitely just change the type itself - we should limit casting and `any` as much as humanly possible

Reply:
But how? this variable _can_ be null at certain times

Reply:
removed this function entirely to bypass this issue. Can be further resolved in the issue to improve the edit profile.